docs/project/2024/teodor.marinescu/index.md

```diff
@@ -0,0 +1,93 @@
+
+### Schematics
+
+Place your KiCAD schematics here.
+
+### Bill of Materials
+
+<!-- Fill out this table with all the hardware components that you might need.
+
+The format is 
+```
+| [Device](link://to/device) | This is used ... | [price](link://to/store) |
+
+```
+
+-->
+
+| Device | Usage | Price |
+|--------|--------|-------|
+| [Rapspberry Pi Pico W](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html) | The microcontroller | [35 RON](https://www.optimusdigital.ro/en/raspberry-pi-boards/12394-raspberry-pi-pico-w.html) |
+| [Breadboard](https://www.yamanelectronics.com/wp-content/uploads/2020/06/basics-of-breadboard.jpg) | The physical base of the project | [10 RON](https://www.optimusdigital.ro/ro/prototipare-breadboard-uri/8-breadboard-830-points.html) |
+| [RGB LED](https://components101.com/diodes/rgb-led-pinout-configuration-circuit-datasheet)| The light bulb |[1,15 Lei](https://www.emag.ro/led-rgb-ws2811-5mm-ai1387/pd/D0WH0GMBM/)|
+| [Sound Sensor](https://components101.com/modules/lm393-sound-detection-sensor-module)| Sensor | [5,67 Lei](https://www.emag.ro/modul-detectare-sunet-cu-microfon-ky-037-compatibil-arduino-ai084-s36/pd/DWY988BBM/?cmpid=93116&gad_source=4)|
+| [Potentiometer](https://components101.com/resistors/potentiometer)| Controller | [2,57 lei](https://ardushop.ro/ro/electronica/193-potentiometru-10k.html?gad_source=4)|
+|[Led 5mm](https://www.farnell.com/datasheets/1498852.pdf)| Red Led light| [0.26 RON](https://www.optimusdigital.ro/ro/optoelectronice-led-uri/700-led-rou-de-3-mm-cu-lentile-transparente.html)|
+| [Male to male jumper wires](https://media.digikey.com/pdf/Data%20Sheets/Digi-Key%20PDFs/Jumper_Wire_Kits.pdf) | Wires | [6.98 RON](https://www.optimusdigital.ro/ro/fire-fire-mufate/888-set-fire-tata-tata-40p-20-cm.html?search_query=fire+tata+tata&results=80) |
+| [Male to male jumper wires](https://media.digikey.com/pdf/Data%20Sheets/Digi-Key%20PDFs/Jumper_Wire_Kits.pdf) | Wires | [6.98 RON](https://www.optimusdigital.ro/ro/fire-fire-mufate/880-fire-colorate-mama-mama-10p-10-cm.html?search_query=fire+mama+mama&results=63) |
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+|[embassy-time](https://embassy.dev/book/dev/time_keeping.html)|Time management library  |Used for time-based operations such as delays |
+|[embassy-rp](https://docs.embassy.dev/embassy-rp/git/rp2040/index.html)| Peripheral access library |Used for initializing and interacting with peripherals |
+|[embassy-executor](https://docs.embassy.dev/embassy-executor/git/std/index.html)|Asynchronous executor for Rust embedded systems| Used for task scheduling and asynchronous programming|
+|[gpio](https://docs.embassy.dev/embassy-stm32/git/stm32c011d6/gpio/index.html)|GPIO manipulation |Used for interacting with GPIO pins |
```

Comment:
This is for STM32, not the Puco.

Reply:
https://www.alexdwilson.dev/learning-in-public/gpio-on-the-raspberry-pi-pico
I used this link as documentation.